src/main/java/com/conveyal/r5/streets/LinkedPointSet.java

```diff
@@ -479,14 +470,46 @@ public PointSetTimes eval (
      *
      * This is a pure function i.e. it has no side effects on the state of the LinkedPointSet instance.
      *
-     * TODO clarify that this can use times or distances, depending on units of the table?
-     * @param distanceTableToVertices a map from integer vertex IDs to distances TODO in what units?
-     * @param distanceTableZone TODO clarify: in fixed or floating degrees etc.
-     * @return A packed array of (pointIndex, distanceMillimeters), or null if there are no reachable points.
+     * @param distanceTableToVertices a map from integer vertex IDs to cumulative distance (in millimeters) accrued
+     *                                traversing the network from the stop to the vertices
+     * @param distanceTableZone the envelope in FIXED POINT DEGREES within which we want to find all points.
+     * @return A packed array of (pointIndex, cost), or null if there are no reachable points.
      */
-    public int[] extendDistanceTableToPoints (TIntIntMap distanceTableToVertices, Envelope distanceTableZone) {
+    public int[] extendDistanceTableToPoints(TIntIntMap distanceTableToVertices, Envelope distanceTableZone) {
+        return extendCostsToPoints(distanceTableToVertices,
+                RoutingVariable.DISTANCE_MILLIMETERS,
+                distanceTableZone,
+                null);
+    }
+
+    /**
+     * Given a table of costs (time or distance) to street vertices from a particular transit stop, create a table of
+     * costs to points in this PointSet from the same transit stop. All points outside the distanceTableZone are
+     * skipped as an optimization. See JavaDoc on the caller (EgressCostTable): this is one of the slowest parts of
+     * building a network.
+     *
+     * This is a pure function i.e. it has no side effects on the state of the LinkedPointSet instance.
+     *
+     * @param sr results of an on-street search from the transit stop
+     * @param distanceTableZone the envelope in FIXED POINT DEGREES within which we want to find all points.
+     * @param egressArea area served by on-demand service from this stop. If null, there are no restrictions on which
+     *                  points can be reached in the egress leg from this stop.
+     * @return A packed array of (pointIndex, cost), or null if there are no reachable points. Cost units match
+     * supplied sr.routingVariable
+     */
+    public int[] extendCostsToPoints(StreetRouter sr, Envelope distanceTableZone, Geometry egressArea) {
```

Comment:
The Envelope is still called "distanceTableZone" even though everything has been changed to "cost".